blockchain/state_migration.go

```diff
@@ -491,31 +489,24 @@ func (bc *BlockChain) StartWarmUp() error {
 		return fmt.Errorf("target cache is nil")
 	}
 	db := state.NewDatabaseWithExistingCache(bc.db, cache)
+	children, err := db.TrieDB().NodeChildren(block.Root())
+	if err != nil {
+		return err
+	}
 
 	bc.quitWarmUp = make(chan struct{})
+	logger.Info("Warm up is started", "blockNum", block.NumberU64(), "root", block.Root().String(), "len(children)", len(children))
 
-	go func() {
-		defer func() {
-			bc.quitWarmUp = nil
-		}()
-
-		children, err := db.TrieDB().NodeChildren(block.Root())
-		if err != nil {
-			logger.Error("Cannot start warming up", "err", err)
-			return
-		}
-
-		logger.Info("Warm up is started", "blockNum", block.NumberU64(), "root", block.Root().String(), "len(children)", len(children))
-
-		resultHashCh := make(chan common.Hash, 10000)
-		resultErrCh := make(chan error)
-
-		for _, child := range children {
-			go bc.concurrentIterateTrie(child, db, resultHashCh, resultErrCh)
-		}
+	resultCh := make(chan struct{}, 10000)
+	errCh := make(chan error)
+	for _, child := range children {
```

Comment:
@ehnuje 
These early children are almost 16. So 16 goroutines are created.
I think this number is quite large. 
So could you consider priority queue in next time like `sync` in sync.go.

```suggestion
	for _, child := range children {
```

Reply:
@KimKyungup If we're running warming-up routines, doesn't it mean that the performance is heavily degraded by the state trie or the given storage trie? In that case, I think it would be better to concentrate the resources to warming-up routine, though it might affect other operations. How do you think?

Reply:
@ehnuje 
When I think about it again, depending on the instance, it seems that 16 goroutines can be too much or too little. It is not easy to change to a queue right away, so let's proceed this way as before.